rubocop-inflector.gemspec

```diff
@@ -33,7 +33,7 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '>= 2.3'
```

Comment:
Note: In this PR, I updated the dependencies that drops support for old version of Rubies.

But if you want to continue to support old version of Rubies,  
I can provide another plan to fix tests.

Which do you like?
```ruby
# spec/spec_helper.rb

begin
  # NOTE: This shared context was introduced in rubocop-rspec 2.15.0 and it requires Ruby >= 2.6.0
  require 'rubocop/rspec/shared_contexts/default_rspec_language_config_context'
  config.include_context 'with default RSpec/Language config', :config
rescue
end
```

Reply:
Thank you for your comment.
Since the RSpec itself has dropped support for Ruby < 2.7, I support dropping old versions of Ruby.